src/main/scala/com/redis/sentinel/SentinelMonitor.scala

```diff
@@ -111,13 +118,17 @@ class SentinelMonitor (address: SentinelAddress, listener: SentinelListener, con
 
 trait SentinelHeartBeater extends Runnable with Log{
   private var running = false
+  private val beatingSentinel = new AtomicBoolean(true) // initialised to true so we don't unnecessarily wake up ops
```

Comment:
sure, keep it. not big deal. what I meant was the usage of this state doesn't reflect its realtime value. in this sense, it's just a stale value which is similar to dirty read. Both are correct but both are not realtime which makes atomic at that level unnecessary.